config/deploy.rb

```diff
@@ -28,7 +28,7 @@
 namespace :deploy do
   desc 'Restart application'
   task :restart do
-    on roles(:app), in: :sequence, wait: 5 do
```

Comment:
`:app` refers to any host running an instance of the application. The `:web` role doesn't include `:app`?